src/code_77480.c

```diff
@@ -202,8 +202,8 @@ void func_800DFC74(void) {
 }
 
 s32 func_800DFCF4(void) {
-    if (D_8010EBB0[0] == 1 && (D_8010EBB0[3] == 6 || D_8010EBB0[3] == 9 || D_8010EBB0[3] == 7 || D_8010EBB0[3] == 4
-                               || D_8010EBB0[3] == 8)) {
+    if (D_8010EBB0.unk_00 == 1 && (D_8010EBB0.unk_03 == 6 || D_8010EBB0.unk_03 == 9 || D_8010EBB0.unk_03 == 7 || D_8010EBB0.unk_03 == 4
+                               || D_8010EBB0.unk_03 == 8)) {
```

Comment:
Perhaps see if there's a nicer way to format this... also put the `{` on its own line since the condition spans multiple